code/modules/client/preferences.dm

```diff
@@ -187,7 +187,13 @@ var/global/list/special_roles = list( //keep synced with the defines BE_* in set
 				dat += "<table width='100%'><tr><td width='24%' valign='top'>"
 
 				if(config.mutant_races)
-					dat += "<b>Species:</b><BR><a href='?_src_=prefs;preference=species;task=input'>[pref_species.name]</a><BR>"
+					if(config.paywall_mutant)
+						if(unlock_content)
+							dat += "<b>Species:</b><BR><a href='?_src_=prefs;preference=species;task=input'>[pref_species.name]</a><BR>"
+						else
+							dat += "<b>Species:</b> Human<BR>"

```

Comment:
Bug, If they already have it set to ligger, and this gets enabled, they will show up as human, but their preference will still be ligger, causing them to automatically change once they buy membership

You should make it selectable, but show some sort of message if paywall_mutant is enabled and they don't have membership. that way they know the option is there, and they can set it, but it does nothing without membership.